RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -10,14 +10,20 @@
     import RxSwift
 #endif
 
```

Comment:
I looked through other parts of the RxSwift codebase and thought an inline comment describing the `MaterializedSequenceResult` was also appropriate, so I've added that also.